app/scripts/views/app.coffee

```diff
@@ -569,10 +564,22 @@ define [
       # TODO @jrwdunham: use the already-in-place @alertDialog for this
       # TODO @jrwdunham @cesine: figure out how i18n/localization works in
       # FieldDB and begin implementing something similar in Dative.
-      console.log ["TODO show some visual thing here using the app view using",
+      console.log ["Pretending the user said Yes. This is dangerous. TODO show some visual confirm dialog here using the app view using",
         "something like http://jqueryui.com/dialog/#modal-confirmation" ,
         message].join ' '
```

Comment:
awesome! i found the code and implemented it. the only problem i see is that your alert dialog doesnt handle cancel events? i will try to add it since the fielddb confirms use it.